connect/settings.py

```diff
@@ -74,11 +74,13 @@ def MIDDLEWARE_CLASSES(self):
             'NAME': os.getenv('DB_NAME'),
             'USER': os.getenv('DB_USER'),
             'PASSWORD': os.getenv('DB_PASSWORD'),
-            'HOST': os.getenv('DB_HOST', '127.0.0.1'),
+            'HOST': os.getenv('DB_HOST', 'localhost'),
             'PORT': os.getenv('DB_PORT', '5432'),
         },
     }
 
+    print(DATABASES)
```

Comment:
``{'default': {'USER': None, 'PORT': '5432', 'PASSWORD': None, 'NAME': None, 'HOST': 'localhost', 'ENGINE': 'django.db.backends.postgresql_psycopg2'}}`` I have this but I am exporting the variables, any idea?